### PR TITLE
Improve PHP version requirement to ^7.1 || ~8.0.0 || ~8.1.0

### PR DESCRIPTION
**Pull request by florianeckerstorfer**
This [came up in the comments](https://github.com/cocur/slugify/pull/299#discussion_r785034646) to a recently closed PR and I would appreciate your input on this.

### CODE REVIEW

**Comment by florianeckerstorfer** [comment]
Added PHP 8.2.x to version range, since the unit tests are currently passing for it

File: composer.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
         }
     ],
     "require": {
-        "php": ">=7.1",
+        "php": "^7.1 || ~8.0.0 || ~8.1.0 || ~8.2.0",
```
Comment (Ocramius) [inline composer.json:22]: Careful with `8.2.0`: I suggest holding off until it reaches RC, around october 2022.

Until then, any BC break is still possible.

**Review by Ocramius — CHANGES_REQUESTED** [review]
Note: you may want to run `composer update --lock`

**Comment by Ocramius** [comment]
Remember `composer.lock`

**Comment by Ocramius** [comment]
@florianeckerstorfer is this closed on purpose?

**Comment by florianeckerstorfer** [comment]
@Ocramius Yes, it took a long while, but I followed your suggestion and excluded 8.2 for now. Merged #313 instead

**Pull request closed without merging**